src/main/java/com/example/bank/service/demo/repository/AccountRepository.java

```diff
@@ -1,11 +1,15 @@
 package com.example.bank.service.demo.repository;
 
 import com.example.bank.service.demo.entity.Account;
+import java.util.List;
 import java.util.Optional;
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.stereotype.Repository;
 
 @Repository
 public interface AccountRepository extends JpaRepository<Account, Long> {
     Optional<Account> getByAccountNumber(String accountNumber);
+
+    @Override
+    List<Account> findAll();
```

Comment:
let's change this method so that we could get all accounts **of a certain User**

Reply:
Done. Sorry for missunderstanding.